libft/sources/numbers/ft_ceil_floor.c

```diff
@@ -0,0 +1,26 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   ft_ceil_floor.c                                    :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: inikulin <inikulin@student.42berlin.de>    +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2024/12/02 18:32:54 by inikulin          #+#    #+#             */
+/*   Updated: 2024/12/02 18:38:25 by inikulin         ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+int	ft_ceil(float a)
+{
+	int	i;
+
+	i = a;
+	if (a > i)
+		return (i + 1);
+	return (i);
+}
+
+int	ft_floor(float a)
+{
+	return ((int)a);
+}
```

Comment:
if you want to handle negative cases:

int result;

result = (int)a;
if (a < 0 && a != result) 
        result -= 1;
    return (result);
    
if not - ignore this.